src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGridItem.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button } from '@openfun/cunningham-react';
+import { DateTime } from 'luxon';
+import { css } from 'styled-components';
+
+import { Box, Icon, StyledLink, Text } from '@/components';
+import { useResponsiveStore } from '@/stores';
+
+import { Doc, LinkReach } from '../../doc-management';
+import { SimpleDocItem } from '../../doc-management/components/items/SimpleDocItem';
+
+import { DocsGridActions } from './DocsGridActions';
+
+type DocsGridItemProps = {
+  doc: Doc;
+};
+export const DocsGridItem = ({ doc }: DocsGridItemProps) => {
+  const { isDesktop } = useResponsiveStore();
+
+  const isPublic = doc.link_reach === LinkReach.PUBLIC;
```

Comment:
It seems to have another state in the design (Atuthenticated ?):
![image](https://github.com/user-attachments/assets/7b645dd2-609d-4ea8-bc95-6fa161cb076c)